Solutions/Illumio Core/Data Connectors/Connector_IllumioCore_CEF.json

```diff
@@ -0,0 +1,121 @@
+{
+    "id": "IllumioCore",
+    "title": "Illumio Core",
+    "publisher": "Illumio",
+    "descriptionMarkdown": "The [Illumio Core](https://www.illumio.com/products/core) data connector provides the capability to ingest Illumio Core logs into Microsoft Sentinel.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on a Kusto Function to work as expected [**IllumioCoreEvent**](https://aka.ms/sentinel-IllumioCore-parser) which is deployed with the Microsoft Sentinel Solution.",

```

Comment:
https://aka.ms/sentinel-IllumioCore-parser - link does not exist

Reply:
Such links are created by Microsoft team